lbrynet/core/Wallet.py

```diff
@@ -1,1325 +0,0 @@
-            remote_height = self.network.get_server_height()
-            updated_blocks_behind = self.network.get_blocks_behind()
-            log.info(
-                'Local Height: %s, remote height: %s, behind: %s',
-                local_height, remote_height, updated_blocks_behind)
-
-            self.blocks_behind = updated_blocks_behind
-            if local_height != remote_height:
-                return
-
-            assert self.blocks_behind == 0
-            self.network.unregister_callback(on_update_callback)
-            log.info("Wallet Loaded")
-            reactor.callFromThread(blockchain_caught_d.callback, True)
-
-        self.network.register_callback(on_update_callback, ['updated'])
-
-        d = defer.succeed(self.wallet.start_threads(self.network))
-        d.addCallback(lambda _: blockchain_caught_d)
-        return d
-
-    # run commands as a defer.succeed,
-    # lbryum commands should be run this way , unless if the command
-    # only makes a lbrum server query, use _run_cmd_as_defer_to_thread()
-    def _run_cmd_as_defer_succeed(self, command_name, *args, **kwargs):
-        cmd_runner = self.get_cmd_runner()
-        cmd = Commands.known_commands[command_name]
-        func = getattr(cmd_runner, cmd.name)
-        return defer.succeed(func(*args, **kwargs))
-
-    # run commands as a deferToThread,  lbryum commands that only make
-    # queries to lbryum server should be run this way
-    # TODO: keep track of running threads and cancel them on `stop`
-    #       otherwise the application will hang, waiting for threads to complete
-    def _run_cmd_as_defer_to_thread(self, command_name, *args, **kwargs):
-        cmd_runner = self.get_cmd_runner()
-        cmd = Commands.known_commands[command_name]
-        func = getattr(cmd_runner, cmd.name)
-        return threads.deferToThread(func, *args, **kwargs)
-
-    def _update_balance(self):
-        accounts = None
-        exclude_claimtrietx = True
-        d = self._run_cmd_as_defer_succeed('getbalance', accounts, exclude_claimtrietx)
-        d.addCallback(
-            lambda result: Decimal(result['confirmed']) + Decimal(result.get('unconfirmed', 0.0)))
-        return d
-
-    def get_max_usable_balance_for_claim(self, claim_name):
-        return self._run_cmd_as_defer_to_thread('get_max_spendable_amount_for_claim', claim_name)
-
-    # Always create and return a brand new address
-    def get_new_address(self, for_change=False, account=None):
-        return defer.succeed(self.wallet.create_new_address(account=account,
-                                                            for_change=for_change))
-
-    # Get the balance of a given address.
-    def get_address_balance(self, address, include_balance=False):
-        c, u, x = self.wallet.get_addr_balance(address)
-        if include_balance is False:
-            return Decimal(float(c) / COIN)
-        else:
-            return Decimal((float(c) + float(u) + float(x)) / COIN)
```

Comment:
^ These kinds of conversions will be inaccurate at some point.  These are a show-stopper to me until corrected.

Reply:
You are commenting on a file that was deleted.